2024-01-14 16:30:06
    [KeyError] - Mapping key not found.
    function: getConfig(('username',))
    traceback: Traceback (most recent call last):
  File "C:\Users\Windows10\AppData\Roaming\Python\Python37\site-packages\igzg\utils.py", line 15, in getConfig
    resList = [config[section][x] for x in configList]
  File "C:\Users\Windows10\AppData\Roaming\Python\Python37\site-packages\igzg\utils.py", line 15, in <listcomp>
    resList = [config[section][x] for x in configList]
  File "D:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\configparser.py", line 1251, in __getitem__
    raise KeyError(key)
KeyError: 'username'

    
2024-01-14 16:31:18
    [KeyError] - Mapping key not found.
    function: getConfig(('username',))
    traceback: Traceback (most recent call last):
  File "C:\Users\Windows10\AppData\Roaming\Python\Python37\site-packages\igzg\utils.py", line 15, in getConfig
    resList = [config[section][x] for x in configList]
  File "C:\Users\Windows10\AppData\Roaming\Python\Python37\site-packages\igzg\utils.py", line 15, in <listcomp>
    resList = [config[section][x] for x in configList]
  File "D:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\configparser.py", line 1251, in __getitem__
    raise KeyError(key)
KeyError: 'username'

    
